doc/examples/applications/plot_fluorescence_nuclear_envelope.py

```diff
@@ -0,0 +1,294 @@
+# Indeed, most scikit-image functions support nD images.
+
+n_z = image_sequence.shape[0]  # number of frames
+
+smooth_seq = filters.gaussian(image_sequence[:, 0, :, :], sigma=(0, 1.5, 1.5))
+thresh_values = [filters.threshold_otsu(s) for s in smooth_seq[:]]
+thresh_seq = [smooth_seq[k, ...] > val for k, val in enumerate(thresh_values)]
+
+#####################################################################
+# Alternatively, we could compute ``thresh_values`` without using a list
+# comprehension, by reshaping ``smooth_seq`` to make it 2D (where the first
+# dimension still corresponds to time points, but the second and last
+# dimension now contains all pixel values), and applying the thresholding
+# function on the image sequence along its second axis:
+#
+# .. code-block:: python
+#
+#     thresh_values = np.apply_along_axis(filters.threshold_otsu,
+#                                         axis=1,
+#                                         arr=smooth_seq.reshape(n_z, -1))
+#
+# We use the following flat structuring element for morphological
+# computations:
+
+footprint = np.stack((np.zeros((3, 3)),
+                      ndi.generate_binary_structure(2, 1),
+                      np.zeros((3, 3))))
+footprint
+
+#####################################################################
+# This way, each frame is processed independently (pixels from consecutive
+# frames are never spatial neighbors).
+
+fill_seq = ndi.binary_fill_holes(thresh_seq, structure=footprint)
+
+#####################################################################
+# When clearing objects which touch the image border, we want to make sure
+# that the bottom (first) and top (last) frames are not considered as borders.
+# In this case, the only relevant border is the edge at the greatest (x, y)
+# values. This can be seen in 3D by running the following code:
+#
+# .. code-block:: python
+#
+#     import plotly.graph_objects as go
+#
+#     sample = fill_seq
+#     (n_Z, n_Y, n_X) = sample.shape
+#     Z, Y, X = np.mgrid[:n_Z, :n_Y, :n_X]
+#
+#     fig = go.Figure(
+#         data=go.Volume(
+#             x=X.flatten(),
+#             y=Y.flatten(),
+#             z=Z.flatten(),
+#             value=sample.flatten(),
+#             opacity=0.5,
+#             slices_z=dict(show=True, locations=[n_z // 2])
+#         )
+#     )
+#     fig.show()
+
+border_mask = np.ones_like(fill_seq)
+border_mask[n_z // 2, -1, -1] = False
+clear_seq = segmentation.clear_border(fill_seq, mask=border_mask)
+
+dilate_seq = morphology.binary_dilation(clear_seq, footprint=footprint)
+erode_seq = morphology.binary_erosion(clear_seq, footprint=footprint)
+mask_seq = dilate_seq.astype(int) - erode_seq.astype(int)
+
+#####################################################################
+# Let us give each mask (corresponding to each time point) a different label,
+# running from 1 to 15.
+
+mask_list = [mask * lab for lab, mask in enumerate(mask_seq, 1)]
+mask_sequence = np.stack(mask_list)
```

Comment:
I think that the inline comment "add one for the background" is misleading, because we don't label the background *in addition* to the other regions...(?)  We just wrote that we wanted labels running from 1 to 15, so it's just that we need to call `np.arange(1, 16)` (and not `np.arange(15)`, which would give values ranging from 0 to 14).